gtest_parallel.py

```diff
@@ -398,9 +436,20 @@ def record_test_time(self, binary, testname, runtime_ms):
   def write_to_file(self, save_file):
     "Write all the times to file."
     try:
-      with open(save_file, "wb") as f:
-        with gzip.GzipFile("", "wb", 9, f) as gzf:
-          cPickle.dump(self.__times, gzf, cPickle.HIGHEST_PROTOCOL)
+      with TestTimes.LockedFile(save_file, 'a+b') as fd:
+        fd.seek(0)
```

Comment:
Yes, it's necessary. When opening file for appending ('a' mode) file's position is at the end of file.